lz4/stream/_stream.c

```diff
@@ -888,13 +889,19 @@ _create_context (PyObject * Py_UNUSED (self), PyObject * args, PyObject * kwds)
       goto abort_now;
     }
 
-  /* Initialize the output buffer */
+  /* Initialize the output buffer
+   *
+   * In out-of-band block size case, use a best-effort strategy for scaling
+   * buffers.
+   */
   if (context->config.direction == COMPRESS)
     {
-      uint32_t store_max_size = _GET_MAX_UINT32(store_comp_size);
+      uint32_t store_max_size = _GET_MAX_UINT32((store_comp_size == 0) ?
+                                                4 :
+                                                store_comp_size);
```

Comment:
This indentation is very weird and hard to read. Can you improve the readability of this ternary operator? I'd even go as far as saying I'd prefer an explicit if/else than this.

Reply:
done